src/count_transformer.jl

```diff
@@ -21,64 +22,64 @@ will be removed. Similarly, the `min_doc_freq` parameter restricts terms in the
 other direction. A value of 0.01 means that only terms that are at least in 1% of
 documents will be included.
 """
-mutable struct BagOfWordsTransformer <: AbstractTextTransformer
+mutable struct CountTransformer <: AbstractTextTransformer
```

Comment:
Mmm. Yes, I see the sense in the new name, but this is a breaking change. In particular, it would require an update to the MLJ model registry and docs. 

We should not conflate a bug fix with breaking change in the same PR. Can we revert the current PR to exclude the name change and if you're sure about the name change, please open a new PR.

Reply:
Yeah good  call - we can push this to some future PR.